src/hooks/api/useAccounts.ts

```diff
@@ -6,20 +6,44 @@ import {
 import { Account } from '@/book/entities';
 import fetcher from './fetcher';
 
-export function useAccount(guid: string): UseQueryResult<Account> {
-  const result = useQuery({
-    queryKey: [...Account.CACHE_KEY, { guid }],
-    queryFn: fetcher(() => Account.findOneBy({ guid }), `/${Account.CACHE_KEY.join('/')}/${guid}`),
-  });
+export function useAccount(guid: string): UseQueryResult<Account | undefined> {
```

Comment:
instead of this being a different hook, it is now a select on top of the `useAccounts` hook. Following https://tkdodo.eu/blog/react-query-data-transformations